apps/condo/domains/common/components/settings/CallCenterSettingsContent.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useState } from 'react'
+import { Col, Row, Typography } from 'antd'
+import get from 'lodash/get'
+
+import { useIntl } from '@condo/next/intl'
+import { useOrganization } from '@condo/next/organization'
+import { TableFiltersContainer } from '@condo/domains/common/components/TableFiltersContainer'
+import Input from '@condo/domains/common/components/antd/Input'
+import { CardsContainer } from '@condo/domains/common/components/Card/CardsContainer'
+import { TicketOrganizationSetting as TicketSetting } from '@condo/domains/ticket/utils/clientSchema'
+import {
+    TicketDeadlineSettingCard,
+} from '@condo/domains/ticket/components/TicketOrganizationDeadline/TicketDeadlineSettingCard'
+
+export const CallCenterSettingsContent: React.FC = () => {
+    const intl = useIntl()
+    const CallCenterTitle = intl.formatMessage({ id: 'CallCenter' })
+    const SearchPlaceholder = intl.formatMessage({ id: 'search' })
```

Comment:
Use a common key format. We don't use just one word ... look at the existing examples

Reply:
they really have no common key format. They are named whatever. And sometimes the same values with different keys are very often duplicated